nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -440,6 +440,7 @@
                                                 <include>${staging.dir}/js/nf/canvas/nf-clipboard.js</include>
                                                 <include>${staging.dir}/js/nf/canvas/nf-custom-ui.js</include>
                                                 <include>${staging.dir}/js/nf/canvas/nf-canvas-utils.js</include>
+                                                <include>${staging.dir}/js/nf/canvas/nf-component-version.js</include>
```

Comment:
I think we can move this down in the order. If you move it below the nf-process-group-configuration.js. then in the nfProcessGroupConfiguration module you can load nfComponentVersion module. If you do this then you can avoid having to inject the nfProcessGroupConfiguration module into the nfComponentVersion module in the nf-canvas-bootstrap.js.